docs/analyzer/user_guide.md

```diff
@@ -254,7 +254,8 @@ analyzer arguments:
   --analyzers ANALYZER [ANALYZER ...]
                         Run analysis only with the analyzers specified.
                         Currently supported analyzers are: clangsa, clang-
-                        tidy.
+                        tidy, cppcheck. Version number can be also specified 
+                        for analyzers to verify them.
```

Comment:
Put this extra text to the argparse "help" attribute too. This documentation is supposed to be a copy-pase of the actual `CodeChecker analyze --help` page.